client/components/badge/README.md

```diff
@@ -24,4 +20,4 @@ class MyComponent extends React.Component {
 
 The following props are available to customize the Badge:
 
-- `type`: `'warning'`, `'success'`, `'info'`, `'info-blue'`, `'error'`
+- `type`: `'warning'`, `'success'`, `'info'`, `'info-blue'`, `'error'`, `'scan-threat'`
```

Comment:
instead of scan-threat we could have a more generic "alert" badge. 

Would the treatment be different from the error badge? 

We should also update the tests to add this case as well as the example that we show in the devdocs.

Reply:
Yeah. From the start, I was a bit hesitant about adding more states/types to this component. What we already have is what most developers are familiar with so adding things specific to a project can make things confusing. So, thanks for noticing this because I wanted to have this conversation with you :)

For me, adding an alert type doesn't add much and makes the API harder to understand. What's the difference between warning and alert? What if in the future we need a green color that matches Jetpack's green but doesn't match the success green? How should we name that color?

What I did in the first place – then I removed the experiment – was to create a JetpackCloudBadge component. Under the hood, it renders a Badge component and its only purpose is to change the colors of the types of the Badge. We could also rename the types to make them better suite JP Cloud.

The usage would be like this:
```
<JPCBadge type="alert/error/threat">X threats</JPCBadge>
```

If we do this, we no longer have to update the Badge component. Therefore, there is no risk of breaking something and making the Badge component more confusing.

What do you think? I'm Ok with adding the alert type to the Badge component, but that solution won't scale.

Reply:
I forgot to say that this problem happened with the LogItem component because under the hood it uses the FoldableCard component which has the highlight property. This property resembles the type property of the Badge component (it accepts similar values).

Reply:
I think we should just use the "error" type badge componet here. Instead of adding a new one. 

I think it works well and the colours are the same as far as I can tell.